digitalfabrik/integreat-app
# 2798: Chat UI components

### Short description

The ui components for the chatbot will be implemented in this pr with mocked data.

### Proposed changes



- add chatbot ui logic
- fixed some external link issues
- add feature flag for chatbot

### Side effects



- none

### TODO
- Implement api logic
- Create unit tests

### Resolved issues



Fixes: #2798 

---

### Testing
- enable feature flag chatbot in build config for integreat
- Choose city "Landeshauptstadt München"
- Open the chatbot
- Type something -> mocked messages should be received
- sessionId should be stored in localStorage
- minimize the chatbot (desktop) 
- maximized the chatbot (desktop) -> Content should maintain
- open and close chatbot (mobile) -> content should maintain
- Chatbot should be shown on all CityContent routes except pois, since its difficult to show a FAB over the map. Even this can be discussed
- Note: the session id will not be cleared atm
